### ISSUE TITLE
Scraping issue with .special name format

### ISSUE THREAD

**Issue by pneshati** [pkscout/metadata.tvshows.thesportsdb.python#5]
Soccer matches not scraping using the .special naming convention as specified in the ReadMe as well as on the SportsDB page. Tried with entries from Premier League, UEFA Champions League, UEFA European Championships and FIFA World Cup matches.

**Comment by pkscout** [comment]
I will need a full Kodi debug log and a screen shot of your directory structure with the game names to troubleshoot.  **DO NOT** enable verbose logging in the scraper.  That will generate an insane amount of logging that shouldn't be needed to look at this issue.

Once you have the log, please post it to a pastebin site and add a link here.  It's fine to add the screenshots here.

**Comment by pneshati** [comment]
Hi, I actually figured this one out through trial and error, but I appreciate you reaching out. In fact, the problem was not with the scraper, but with the SportsDB event entries. It looks like the English Premier League games without round/episode numbers refuse to scrape. I went ahead and adjusted them to fit the correct matchweek.

One issue I have encountered is with the 'fuzzy filename matching' mentioned in the .special naming convention in regards to seasons or tournaments; often, the scraper seems to struggle and returns a different match from the same matchweek or tournament round. Sometimes re-scraping will provide the correct match, although it is inconsistent and sometimes does not.

Thanks!
